### PR TITLE
Adding Social Share Example

### PR DESCRIPTION
**Pull request by paul-matthews**
* Covers all regular examples

### CODE REVIEW

**Comment by sebastianbenz** [comment]
Has the extension been already released to prod as the validation in the build fails?

**Comment by paul-matthews** [comment]
Please don't merge this until amp-social-share is published (merged to release). https://github.com/ampproject/amphtml/commit/519b3e8877a672d40461beb15b69181dc83b7526

Should be later this week.
https://github.com/ampproject/amphtml/commits/release

File: src/20_Components/amp-social-share.html [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+-->
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <!-- #### Setup -->
+  <!--
+    Ensure you've configured your canonical URL.
+
+    Import the `amp-social-share` component in the header.
```
Comment (sebastianbenz) [inline src/20_Components/amp-social-share.html]: can you move `Import the `amp-social-share` component in the header.` into it's own comment block above the import statement. This makes sure the sections are properly aligned.

File: src/20_Components/amp-social-share.html [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+</head>
+<body>
+
+  <!-- #### Basic Usage -->
+  <!--
+    Embed the `amp-social-share` widget choosing the a share type from the [supported types](https://github.com/ampproject/amphtml/blob/master/extensions/amp-social-share/amp-social-share.md#types), ensuring you use the relevant width and height.
```
Comment (sebastianbenz) [inline src/20_Components/amp-social-share.html]: typo "the a" share type

**Comment by sebastianbenz** [comment]
If it is already available as an experiment you can add the following block:

```
<!---{
  "experiment": true,
  "component": "amp-experimenal-component"
}--->
```
This will exclude it from validation and show a card on how to enable experiments.

**Pull request merged**